website/docs/how-to/how-to-clone-environment.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: How to clone an environment
+---
+
+:::info availability
+
+Environment cloning is planned to be released in **Unleash 4.18**.
```

Comment:
Unsure about this, just a guess. We might want to include it right away in 4.17.

Reply:
Yeah, nice suggestion, we can make it more narrow when we know better